easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -18,12 +18,19 @@ builddependencies = [('zlib', '1.2.8')]
 preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
 
-# make sure that system libraries are also considered by ld
-configopts = '--with-sysroot=/'
+# make sure that system libraries are also considered by ld and ld.gold is also built
+configopts = '--with-sysroot=/ --enable-gold --enable-ld=default '
 
-binlist = ['addr2line', 'ar', 'as', 'c++filt', 'elfedit', 'gprof', 'ld', 'ld.bfd', 'nm',
-           'objcopy', 'objdump', 'ranlib', 'readelf', 'size', 'strings', 'strip' ]
+# Ensure PIC
+buildopts = 'CFLAGS="-fPIC"'
+
+# If you require libiberty uncomment the following
+#postinstallcmds = ["cp libiberty/libiberty.a %(installdir)s/lib/; cp include/libiberty.h %(installdir)s/include/",
```

Comment:
This has popped up again in https://github.com/hpcugent/easybuild-easyblocks/issues/969#issuecomment-231339463

Not sure things have progressed within binutils since then, but from  this thread including libiberty from binutils is non-standard so we should maybe think hard about it.